webcompat/templates/layout.html

```diff
@@ -19,7 +19,8 @@
 {% include "shared/footer.html" %}
 {%- if config.PRODUCTION or config.STAGING -%}
 <!-- Google Analytics -->
-<script src="{{ url_for('static', filename='dist/ga.js') }}"></script>
+<script type="module" src="{{ url_for('static', filename='dist/ga.js')| bust_cache}}"></script>
+<script nomodule src="{{ url_for('static', filename='dist/ga.es5.js')| bust_cache }}"></script>
```

Comment:
ga.js doesn't have any es6 code, but in case this ever changes, I've added this es5 file bundle for the older browsers